agents/ls-clang/src/main/resources/installers/1.0.0/org.eclipse.che.ls.clang.script.sh

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+unset PACKAGES
+unset SUDO
+command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
+command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
+test "$(id -u)" = 0 || SUDO="sudo -E"
+
+AGENT_BINARIES_URI=https://codenvy.com/update/repository/public/download/org.eclipse.che.ls.json.binaries
+CHE_DIR=$HOME/che
+LS_DIR=${CHE_DIR}/ls-clangd
+LS_LAUNCHER=${LS_DIR}/launch.sh
+
+if [ -f /etc/centos-release ]; then
+    FILE="/etc/centos-release"
+    LINUX_TYPE=$(cat $FILE | awk '{print $1}')
+ elif [ -f /etc/redhat-release ]; then
+    FILE="/etc/redhat-release"
+    LINUX_TYPE=$(cat $FILE | cut -c 1-8)
+ else
+    FILE="/etc/os-release"
+    LINUX_TYPE=$(cat $FILE | grep ^ID= | tr '[:upper:]' '[:lower:]')
+    LINUX_VERSION=$(cat $FILE | grep ^VERSION_ID=)
+fi
+
+MACHINE_TYPE=$(uname -m)
+
+mkdir -p ${CHE_DIR}
+mkdir -p ${LS_DIR}
+
+#########################
+#### Install packages ###
+#########################
+#
+## Red Hat Enterprise Linux 7
+#############################
+#if echo ${LINUX_TYPE} | grep -qi "rhel"; then
+#    test "${PACKAGES}" = "" || {
+#        ${SUDO} yum install ${PACKAGES};
+#    }
+#
+#    command -v nodejs >/dev/null 2>&1 || {
+#        curl --silent --location https://rpm.nodesource.com/setup_6.x | ${SUDO} bash -;
+#        ${SUDO} yum -y install nodejs;
+#    }
+#
+## Red Hat Enterprise Linux 6
+#############################
```

Comment:
Ideally we have to support all possible linux distributions

Reply:
yes, i will think about a way to install it through the script